### PR TITLE
Update to using Kilo

### PR DESCRIPTION
**Pull request by ramereth**
This is still a work in progress and depends on #25.

### CODE REVIEW

File: Berksfile [hunk 1]
```diff
@@ -2,39 +2,32 @@ source 'https://supermarket.chef.io'
...
-cookbook 'apache2', '< 2.0.0'
 cookbook 'base', git: 'git@github.com:osuosl-cookbooks/base'
-cookbook 'database', '>= 2.0.0'
 cookbook 'firewall', git: 'git@github.com:osuosl-cookbooks/firewall'
 cookbook 'modules', git: 'git@github.com:osuosl-cookbooks/modules-cookbook'
 cookbook 'monitoring', git: 'git@github.com:osuosl-cookbooks/monitoring'
 cookbook 'munin'
-cookbook 'nagios', git: 'git@github.com:osuosl-cookbooks/nagios'
+cookbook 'osl-nagios', git: 'git@github.com:osuosl-cookbooks/osl-nagios'
+cookbook 'osl-nrpe', git: 'git@github.com:osuosl-cookbooks/osl-nrpe'
```
Comment (armiller) [inline Berksfile]: I guess this is blocked by me for the time being?

File: .kitchen.cloud.yml [hunk 2]
```diff
@@ -78,18 +74,21 @@ suites:
       - recipe[osl-openstack::default]
   - name: controller
     run_list:
+      - role[openstack_tk]
       - recipe[osl-openstack::ops_database]
       - recipe[osl-openstack::controller]
       - recipe[openstack-integration-test::setup]
-      - recipe[scl]
+      #- recipe[scl]
```
Comment (jordane) [inline .kitchen.cloud.yml]: Is there a reason this is commented out instead of removed?

Comment (ramereth) [inline .kitchen.cloud.yml]: This pull request is still a work-in-progress. I will likely remove it in the final form.

File: Berksfile (hunk 1 above)
Comment (ramereth) [inline Berksfile]: My intention is that this PR is depended on that but I don't expect this to be ready for deploying to production for another month or so. So I'm not that concerned.

**Comment by ramereth** [comment]
I'm closing this in favor of a liberty branch I'm currently working on.

**Pull request closed without merging**